config/config.exs

```diff
@@ -43,7 +43,8 @@ config :glimesh,
 
 config :waffle,
   storage: Waffle.Storage.Local,
-  storage_dir: "uploads"
+  storage_dir: "uploads",
+  version_timeout: 4_000
```

Comment:
The driver for 4 seconds is that the default write timeout for the HTTP lib Glimesh/janus-ftl-player is 5 seconds.